tutorials/scripting/c_sharp/c_sharp_signals.rst

```diff
@@ -30,8 +36,13 @@ In addition, you can always access signal names associated with a node type thro
 .. warning::
 
     While all engine signals connected as events are automatically disconnected when nodes are freed, custom
-    signals aren't. Meaning that: you will need to manually disconnect (using ``-=``) all the custom signals you
-    connected as C# events (using ``+=``).
+    signals connected using ``+=`` aren't. Meaning that: you will need to manually disconnect (using ``-=``)
```

Comment:
```suggestion
    signals connected using ``+=`` aren't. Meaning that you will need to manually disconnect (using ``-=``)
```

Reply:
To me, without the `:`, it's a conversational sentence fragment, but it doesn't read all that well with the `:`. Maybe `Meaning:` would sound ok to me. I think `This means` is safe/natural/boring (and pushed that for now), but I'm open to whatever.